gnc/sub8_perception/nodes/buoy_2d.py

```diff
@@ -60,9 +51,15 @@ def __init__(self):
             'green':deque()
         }
         self.buoys = {
-            'green': '/color/buoy/green',
-            'red': '/color/buoy/red',
-            'yellow': '/color/buoy/yellow',
+            # Boost classifier paths
```

Comment:
Comment 55-57 and uncomment 60-62 for thresholding. The threshold values are in sub8_launch/config/default_color_calibration.yaml. **NOTE:** the channel marker threshold values there aren't used yet.